docs/install/Fedora.md

```diff
@@ -14,7 +14,7 @@ dnf install atomic
 ##Make
 On Fedora, You need to install required build dependencies
 ```
-yum-builddep atomic
+#yum-builddep atomic
```

Comment:
Please send email to dwalsh@redhat.com and I can help you.  I am closing this issue.